db/migration/202009171251-init-database.go

```diff
@@ -0,0 +1,88 @@
+package migration
+
+import migrate "github.com/rubenv/sql-migrate"
+
+func Get202009171251() *migrate.Migration {
+	return &migrate.Migration{
+		Id: "202009171251-init-database",
+		Up: []string{`
+SELECT 'CREATE DATABASE tinkerbell' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'tinkerbell');
+
+SET ROLE tinkerbell;
```

Comment:
this should probably happen before the `SELECT 'CREATE...'` if possible.

Reply:
Done